rollup.config.js

```diff
@@ -22,7 +22,18 @@ const getBabelOptions = ({ useESModules }) => ({
   plugins: [['@babel/transform-runtime', { useESModules }]],
 });
 
-const matchSnapshot = process.env.SNAPSHOT === 'match';
+const snapshotArgs = (() => {
+  const shouldMatch = process.env.SNAPSHOT === 'match';
+
+  if (!shouldMatch) {
+    return {};
+  }
+
+  return {
+    matchSnapshot: true,
+    threshold: 1000,
```

Comment:
`threshold` could be just passed. It's ignored with `matchSnapshot === false`.

Reply:
I am happy to keep it as is because it is clear - but you're more than welcome to change it 👍